src/containers/pages/SMC/Map/tests/index.test.tsx

```diff
@@ -62,6 +62,7 @@ describe('components/SMC Reports/SMCReportingMap', () => {
   beforeEach(() => {
```

Comment:
Are these tests sufficiently ensuring that this code change works as advertized?

Reply:
@moshthepitt mounting our components on tests shows no data until we update the mounted component, and at that moment it is still not possible to catch the flushing error. This makes it hard to have a test that will ensure no error is flushed on the page before a map appears.

**Before**

https://user-images.githubusercontent.com/26719807/124737952-21dd5400-df21-11eb-969c-5da0fda4bdb0.mp4



**After** 

https://user-images.githubusercontent.com/26719807/124737988-299cf880-df21-11eb-88cd-145bdadcf60c.mp4

Reply:
in the "before" video it appears that some component was rendered just before the map.  If this component was mocked or otherwise stubbed, would a test that asserts it is not rendered/called suffice?